rust/routee-compass/src/app/compass/config/traversal_model/speed_grade_energy_model_builder.rs

```diff
@@ -8,9 +9,10 @@ use crate::app::compass::config::{
 };
 use routee_compass_core::model::traversal::traversal_model::TraversalModel;
 use routee_compass_core::util::unit::{
-    DistanceUnit, EnergyRate, EnergyRateUnit, GradeUnit, SpeedUnit, TimeUnit,
+    grade_unit, DistanceUnit, EnergyRate, EnergyRateUnit, GradeUnit, SpeedUnit, TimeUnit,
```

Comment:
let's remove this weird import i'm guessing rust-analyzer suggested, that looks as though it gets shadowed by variables in the code anyway

Reply:
Yeah that's a weird one. I just found out that you can do `cargo fix` and it will automagically remove unused import and apply other linting fixes. I just applied it here to pull out some more imports that aren't being used.